### ISSUE TITLE
how can I use closeEventCallback ?

### ISSUE THREAD

**Issue by yanzheng000** [stomp-js/stomp-websocket#25]
I saw closeEventCallback in [https://github.com/stomp-js/stomp-websocket/pull/23](url)
but in index.d.ts 
there are only 2 ways :
`connect(headers: StompHeaders, connectCallback: (frame?: Frame) => any, errorCallback?: (error: string) => any): any;
    connect(login: string, passcode: string, connectCallback: (frame?: Frame) => any, errorCallback?: (error: string) => any, host?: string): any;`

How to use closeEventCallback ? Do I need to change index.d.ts ?

**Comment by kum-deepak** [comment]
Yes that will be the case. Will you be able to make the change and send a pull request?

Otherwise I will do it in the weekend.

**Comment by kum-deepak** [comment]
Planning for next version of stompjs is on the way. Your participation will be really appreciated, please join at https://github.com/stomp-js/stompjs/issues/1
